oeps/architectural-decisions/oep-0064-arch-mobile-application-codebase-modernization.rst

```diff
@@ -0,0 +1,99 @@
+
+.. contents::
+  :local:
+  :depth: 1
+
+Abstract
+********
+
+As part of the discovery effort done for a funded contribution (FC-0011) we are recommending that the Open edX platform adopt newly built iOS and Android applications as the new default mobile applications for the platform moving forward, starting with the Quince release. These new applications are rebuilt with Model View ViewModel (MVVM) architectural patterns and are fully rebuilt in languages native to the mobile platforms (Swift and Kotlin, respectively). These new code bases should facilitate faster development cycles, improved pluggability and community contribution rates. As of July 2023, not all functionality is 100% identical to the existing mobile applications, but efforts are underway to improve pluggability and close common community requirements for the mobile experience.
+
+Motivation
+**********
+
+The motivation for this change is to drive adoption, contribution, and improvement of the Open edX platform mobile experiences. The existing mobile applications were authored in 2015 and are not broadly used across the Open edX ecosystem today. Improvements to architectural patterns, development languages, and mobile operating systems have transformed mobile development in ways that make it difficult for our existing code bases to adopt quickly. The Raccoon Gang mobile team rebuilt and open sourced new ios and android applications that form a strong architectural basis on which to continue community development of the mobile applications. 
+
+A robust mobile experience makes learning opportunities more accessible whether that is for learners who don't have ready access to personal computers, or need to fit learning into their schedules around work and family commitments.
+
+
+Specification 
+************* 
+
+The existing mobile repositories are maintained by edX / 2U and can be accessed here:
+
+* https://github.com/openedx/edx-app-ios
+* https://github.com/openedx/edx-app-android
+
+* Support for the existing mobile applications will continue through the Redwood release. Starting with the Quince release, we will designate the new repositories as the default Open edX mobile applications to encourage adoption and use of the new applications for community team members.
+* Once available, we will link to documentation within the new repositories to describe transition options for community members migrating from existing builds of the older codebases.
+* After the publication of the Quince release we will move the original repositories or the openedx-unsupported github organization to help make clear that these are no longer the default mobile application builds moving forward. A corresponding DEPR ticket will be linked to this OEP once it exists. 
+
+The new mobile repositories are maintained by Raccoon Gang and can be accessed here:
```

Comment:
@marcotuts or @e0d, can you review the formatting of this section? I would suggest a fix ,but I'm not sure if this is meant to be one bullet list or multiple.

Here's how it currently renders: https://open-edx-proposals--496.org.readthedocs.build/en/496/architectural-decisions/oep-0064-arch-mobile-application-codebase-modernization.html

![image](https://github.com/openedx/open-edx-proposals/assets/3628148/df57e147-0f88-4c60-a90f-4342c7bb8801)

Reply:
Thanks for flagging Kyle, I didn't notice this so at some point I must have broken this. I can open a new PR for formatting fixes to this.